### ISSUE TITLE
Need to support both replicationcontrollers and replicasets

### ISSUE THREAD

**Issue by therc** [kubernetes/dashboard#384]
I am using the new Deployment stuff and they create replicasets, not replicationcontrollers. Thus it's as if they don't exist. :-( I suspect this would easily reuse the vast majority of the code for RCs.

**Comment by bryk** [comment]
You're correct. I'm afraid this will not come into our current release, but next one. We'll be doing releases often, though.

**Comment by therc** [comment]
What are the planned release dates for both?
RC and RS: the deprecated one and the one that doesn't work yet. :-)

**Comment by bryk** [comment]
The first release is March 7th. Next releases will happen on fairly regular basis or when we have a new feature implemented.

**Comment by bryk** [comment]
By the way, if you want to have this done faster, just contribute :) We're always happy to have more people in the project.

**Comment by therc** [comment]
I'd love to, but I don't know any of this Node stuff, plus my current priority is helping with getting GPU scheduling support in Kubernetes. :-(

**Comment by dpratt** [comment]
I'd like to chime in for this issue as well - we almost exclusively use Deployments in our cluster, which makes the dashboard not terribly useful, unfortunately.

**Comment by c1freitas** [comment]
+1 We are only using Deployments at this point.

**Comment by bryk** [comment]
Update: supporting Deployments/ReplicaSets/DaemonSets/etc is high priority for us at the moment. We have concrete plans on how to do do this. Roadmaps will be posted here in a matter of days. If you want to help us with this, let's talk :)

**Comment by Morriz** [comment]
+1

**Comment by oO-Zam** [comment]
We've also migrated everything to Deployments, and happen to be using Node/Angular/Angular-Material for our internal Admin UI as well. Open to contributing.

**Comment by bryk** [comment]
@oO-Zam Can you help us with work on Deployment list/status/detail pages?

**Comment by christopher5106** [comment]
+1

**Comment by bryk** [comment]
Status update: we have all our designs unblocked and as of now I'm writing frontend for ReplicaSets + Replication Controllers. This will be released soon.

**Comment by Stono** [comment]
Hey guys, 
I still can't see stuff I've deployed from yaml in the dashboard?
Any updates?

**Comment by maciaszczykm** [comment]
Hi @Stono, could you please provide us more details? What are you deploying? What is your setup? It would be good idea to create separate issue, so we can track it.

**Comment by bryk** [comment]
What's the version of the UI you use? Deployments are in 1.1.0.

**Comment by Stono** [comment]
Hi guys, apologies for being a complete amateur here but I'm very new to kubernetes, I did a "next next next" setup on google cloud platform, creating a cluster which appears to have an old version of the dashboard installed

```
 > kubectl get replicationcontrollers --namespace kube-system
NAME                          DESIRED   CURRENT   AGE
kube-dns-v11                  1         1         1d
kubernetes-dashboard-v1.0.1   1         1         17h
l7-lb-controller-v0.6.0       1         1         1d
```

Any idea how to go about updating it?

**Comment by bryk** [comment]
By the end of the week you should log in to the Cloud Console and see "upgrade available" next to your cluster in the list. Or, alternatively, do the same via CLI. When you do this, you'll have a brand new version of the UI :)

**Comment by Stono** [comment]
Ahhh awesome.
I shall wait patiently then! 

Thanks for all your efforts guys, stuff like this makes my job a hell of a lot simpler 👍
